<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskAdapterGui</class>
 <widget class="QDialog" name="TaskAdapterGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="configFileLineEdit">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>70</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveCurrentButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save current to file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Config file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="launchButton">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Launch Task</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="taskActive">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>15</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="taskRunning">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>15</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="taskComplete">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>15</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>22</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Active:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>19</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Running:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Complete:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previewFileButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previewBlankButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Blank</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview from:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previewCurrentButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Current</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>91</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scene roots:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="sceneRootList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>91</width>
      <height>141</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>150</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>269</width>
      <height>139</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
